extensions/amp-a4a/0.1/a4a-variable-source.js

```diff
@@ -140,7 +140,7 @@ export class A4AVariableSource extends VariableSource {
    *     requested attributes, then nothing will be included in the array
    *     for that element.
    */
-  htmlAttrBinding_(cssSelector, ...attributeNames) {
+  htmlAttributeBinding_(cssSelector, ...attributeNames) {
```

Comment:
oops. meant to remove prefix `...` from `...attributeNames` 😄 
See what we did [here](https://github.com/ampproject/amphtml/blob/master/src/layout-rect.js#L124)

Reply:
Ah, I see. Done. Thanks!